free-programming-books-tr.md

```diff
@@ -25,12 +25,14 @@
 
 ### Android
 
+* [Android Dersleri] (httpS://umiitkose.com/android/)
```

Comment:
Could you remove the space between the name and link?
Also, please use a lower case S in the link. (Figures I'd comment on it since there's another change anyway.)

```suggestion
* [Android Dersleri](https://umiitkose.com/android/)
```